### PR TITLE
added audio example notebook

### PR DESCRIPTION
**Pull request by jwmandeville**
<summary of the change, relevant motivation, and context>

Change type:

- [ ] Bug fix
- [ ] Feature
- [X] Documentation

Checklist:

- [x] My code follows the style guidelines of this [project](https://docs.google.com/document/d/1jckZJe0CrWyF-IjxoO2OfBKAyKLC3Yk9Xr7UmOLBETA/edit?usp=sharing)
- [x] I have performed a self-review of my own code
- [x] I have commented my code and added docstrings to all exposed functions and classes
- [x] I have made corresponding changes to the documentation
- [x] Any relevant changes to third party licenses have been updated in the README

### CODE REVIEW

**Review by timesler — APPROVED** [review]
Fantastic work. I made two very minor changes:
* Changed "XT-training" "Xt-training" in the heading
* Changed the SKDataset description to this:
    `SKDataset` makes one major change: it returns the sample and the label as a single tuple (x,y), but also returns the label alone. This is because the `fit` method of sklearn models expects both x and y together, but pytorch handles the input sample and the label separately, in the `forward` method and loss function, respectively. Let's double check that this is all working as intended.
